server/services/v1/database/query_runner.go

```diff
@@ -247,7 +248,8 @@ func (runner *UpdateQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 
 		newData := internal.NewTableDataWithTS(row.Data.CreatedAt, ts, merged)
 		newData.SetVersion(coll.GetVersion())
-		// as we have merged the data, it is safe to call replace
+
+		request.IncrementWrittenBytes(ctx, len(merged))
```

Comment:
We need to update it for read queries that are scanning a lot of documents.

Reply:
Yes, we will need to do two things here for the Update:
- count the number of bytes read to fetch the data to be updated (read units consumed)
- count the amount of data written back (write units consumed)